cdap-docs/_common/common-build.sh

```diff
@@ -131,8 +131,11 @@ function usage() {
 
 function clean() {
   cd $SCRIPT_PATH
-  rm -rf $SCRIPT_PATH/$BUILD
-  mkdir $SCRIPT_PATH/$BUILD
+  if [ -d "${SCRIPT_PATH}/${BUILD}" ]
+  then
+    rm -rf $SCRIPT_PATH/$BUILD
```

Comment:
convention is to put braces around all variables always: ``${SCRIPT_PATH}/${BUILD}``